.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  rustfmt:
+    docker:
+      - image: circleci/rust:latest
```

Comment:
Is `latest` like latest nightly or latest stable? Something to watch out for if we pin to a single version elsewhere.